analyses/compare-rnaseq-pipelines/README.md

```diff
@@ -0,0 +1,100 @@
+# Compare RNA-Seq workflows
+
+**Contents**
+
+- [Purpose](#purpose)
+- [Usage](#usage)
+  - [Correlation Coefficient](#correlation-coefficient)
+  - [Software dependencies](#software-dependencies)
+
+## Purpose
+The pipeline-comparison module performs a comparative analysis of varying RNA-Seq workflows, offering insight into the correlation coefficient across workflows for counts at gene-level. Currently, the comparison is performed on the PPTC, KidsFirst, and JAX workflows.
+Varieties of comparison include (a) comparison of the full gene set, at gene level (b) partial gene set, only homologous to mouse genome (c) comparison of the full gene set, at gene level with threshold on the TPM value (d) partial gene set, only homologous to mouse genome with thresold on the TPM value
+
+## Usage
+The input files must be two RNA-Seq TPM gene expression tables in tab separated formats (.tsv) 
+Other input must also include the names of the workflows to compare and a file containing a list of genes homologous to mouse
+
+
+### Correlation Coefficient
+Generates the correlation matrix along with standard and probably error of the correlation coefficient
```

Comment:
```suggestion
Generates the correlation matrix along with standard and probable error of the correlation coefficient
```
@sangeetashukla,  Would be helpful if you could include information on how you calculated standard and probable errors in the README for users. Currently buried deep in the code

Reply:
@ewafula Makes sense. I will update the readme with that information.

Reply:
@ewafula I made some updates to the README so it has more information on the significance and interpretation of the correlation coefficient and errors.